dist/init/linux-systemd/README.md

```diff
@@ -6,26 +6,153 @@ if you have any questions. Feel free to prepend to your question
 the username of whoever touched the file most recently, for example
 `@wmark re systemd: …`.
 
-The provided file should work with systemd version 219 or later. It might work with earlier versions.
+The provided file should work with systemd version 229 or later. It might work with earlier versions.
 The easiest way to check your systemd version is to look at the version of the installed package
 (e.g. 'sudo yum info systemd' on RedHat/Fedora systems).
 
-## Instructions
-
 We will assume the following:
 
-* that you want to run caddy as user `www-data` and group `www-data`, with UID and GID 33
+* that you want to run caddy as user `www-data` and group `www-data`
 * you are working from a non-root user account that can use 'sudo' to execute commands as root
 
 Adjust as necessary or according to your preferences.
 
-First, put the caddy binary in the system wide binary directory and give it
-appropriate ownership and permissions:
+## Install
+
+There's a full caddy install that works with systemd over at https://git.daplie.com/Daplie/caddy-installer
```

Comment:
Goes out of this repo. Please remove this.

(I've glanced of the referenced **bash** file and don't like it.)